src/types.rs

```diff
@@ -655,3 +656,63 @@ pub fn creation_time(creation_time: u64) -> u64 {
         creation_time
     }
 }
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize)]
+pub enum DataType {
+    Binary = 0x000000,
+    Text = 0x000001,
+    Image = 0x00000D,
+    TempoCpil = 0x000015,
```

Comment:
Sorry, what type is this?

Reply:
Thanks for the review !
Those are the type used to format data in an `ilst` element. They are pretty obvious but `TempoCpil`which I have no idea what it is about. I took it from here: https://xhelmboyx.tripod.com/formats/mp4-layout.txt.
`Binary` is raw vector of u8, `Image`is the same but you know it is an image, `Text`should be valid utf_8 data.

Reply:
Yeah, I was referring to the `TempoCpil ` type, but thanks for the clarification! Looks like it's referring to `tmpo/cpil flag = 0x000015`.

Reply:
I have no clue of what it is for and left it for sake of completeness.